Source/Reporting/Domain/Management/DataCollectors/Registration/RegisterDataCollector.cs

```diff
@@ -24,6 +24,7 @@ public class RegisterDataCollector : ICommand
 
         public Region Region { get; set; }
         public District District { get; set; }
+        public Village Village { get; set; }
```

Comment:
Does this make any change?

Reply:
Or this i maybe my change that I have (hopefully) reverted?

Reply:
You're right, this hung around from when we were trying to change the event. I'm removing it